src/bundler.zig

```diff
@@ -1435,12 +1435,18 @@ pub const Bundler = struct {
                                     }, prop.key.?.loc),
                                     .value = prop.value.?,
                                 };
+                                var maybe_quoted = name;
+                                if (strings.containsChar(name, '-')) {
```

Comment:
Is there already a function that checks if a string is a valid variable name or do I need to write one ?

Reply:
yeah, `MutableString.ensureValidIdentifier`. It will check if it's a valid identifier, and create a new one with the non-valid characters replaced by `_` if it is not valid